src/item.py

```diff
@@ -0,0 +1,13 @@
+class Item:
+    def __init__(self, name, description):
+        self.name = name
+        self.description = description
+
+    def get_name(self):
+        return self.name
+
+    def __str__(self):
+        return f"\nItem: {self.name}, Description: {self.description}\n"
```

Comment:
Nice work on using str and repl, saves lots of code blocks for user output and dev debugging.